release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -150,6 +150,13 @@ With this release, you can now require your workloads to use a specific security
 
 For more information, see xref:../authentication/managing-security-context-constraints.adoc#security-context-constraints-requiring_configuring-internal-oauth[Configuring a workload to require a specific SCC].
 
+[id="ocp-4-14-auth-psa-exemptions"]
+==== Namespaces exempt from pod security admission
```

Comment:
Should the abbreviation PSA be used also? Also I see "Pod Security Admission" in caps in some of our docs. Example [here](https://docs.openshift.com/container-platform/4.13/rest_api/operatorhub_apis/catalogsource-operators-coreos-com-v1alpha1.html#:~:text=to%20allow%20the%20pod%20to%20run%20in%20Pod%20Security%20Admission)

Reply:
We don't abbreviate it as PSA officially in the docs anywhere (that I'm aware of). Same with the capitalization. This is the main page about pod security admission, it is spelled out and lowercase: https://docs.openshift.com/container-platform/4.13/authentication/understanding-and-managing-pod-security-admission.html

The example you posted is from the API docs, which are auto-generated from engineering content, so that isn't something that always uses proper wording or capitalization

Reply:
NP. Thanks for clarifying!